packages/typespec-ts/test/commands/cadl-ranch-list.js

```diff
@@ -562,6 +562,10 @@ export const modularTsps = [
   {
     outputPath: "client/structure/client-operation-group",
     inputPath: "client/structure/client-operation-group"
+  },
+  {
```

Comment:
@v-jiaodi Could you disable the routes testing considering we have issue:https://github.com/Azure/autorest.typescript/issues/2866?